docs/external-interop/origin_avalanche/summary.md

```diff
@@ -0,0 +1,25 @@
+# Sending messages from Avalanche
+
+This section of the design document is concerned with messages originating from Avalanche. In this section, we detail how external EVM chains authenticate messages from Avalanche and the infrastructure for sending messages and maintaining  trust.
+
+## Message protocol
+
+Communication between Avalanche L1 (the so-called _"internal interoperability"_) uses the [ICM protocol](../../../icm-contracts/contracts/teleporter/README.md)  which is handled by a set of `Teleporter` contracts. Further protocols can be built on top of `Teleporter`, but this is  out of scope for this design doc. The goal of external interoperability is to support `Teleporter` contracts on external EVM  chains to leverage existing protocols. This will require changes to the `TeleporterMessenger` contracts which will be  deployed on external EVM chains; these changes are detailed in the [TeleporterMessesenger changes](teleporter_messenger_changes.md) section.
+
+The primary difference between internal and external interoperability is how message authentication is accomplished. For internal interoperability, this was achieved used a pre-compiled smart contract called `Warp`. An alternative solution is necessary for external EVM chains because the `Warp` pre-compile is specific to Avalanche.
+
+This will also require changing the relayers. They will need clients for external EVM chains and will need to call different contract functions and provide different data as well. There also new duties required of relayers other than just sending messages that will be required. This is further describe in the [Relayer changes](relayer_changes.md) section.
+
+## Root of trust
+
+The Avalanche P-chain is the root of trust when authenticating messages from the Avalanche C-chain or an Avalanche L1. Specifically, all authentication proofs should derive their authority from an initial  P-chain validator set known to the external EVM chain. While the publication of this initial set on an external EVM chain may be done anyone, it's validity is publicly auditable and thus a trustless procedure.
```

Comment:
A couple nits:
1. I think we're misusing "P-Chain validator sets" here. It will be important to distinguish between a "the L1 validator set registered on the P-Chain" and "the P-Chain validator set".
2. Since this is meant to be external facing documentation, we should use "P-Chain" and "C-Chain" capitalization, which we try to be standardized with across the board.